configs/alcf/config.20171031.tenYearTest.GMPAS-IAF.T62_oEC60to30v3wLI.60layer.theta

```diff
@@ -112,67 +118,3 @@ endYear = 9999
 startYear = 1
 endYear = 9999
 
-[oceanObservations]
-## options related to ocean observations with which the results will be compared
-
-# directory where ocean observations are stored
-baseDirectory = /lus/theta-fs0/projects/ClimateEnergy_2/diagnostics/observations/Ocean
-
-[seaIceObservations]
-## options related to sea ice observations with which the results will be
-## compared
-
-# directory where sea ice observations are stored
-baseDirectory = /lus/theta-fs0/projects/ClimateEnergy_2/diagnostics/observations/SeaIce
-
-[timeSeriesSeaIceAreaVol]
-## options related to plotting time series of sea ice area and volume
-
-# plot on polar plot
-polarPlot = False
-
-[regions]
-# Directory containing mask files for ocean basins and ice shelves
-regionMaskDirectory = /lus/theta-fs0/projects/ClimateEnergy_2/diagnostics/mpas_analysis/region_masks
-
```

Comment:
why is the section below removed from this config file (same goes for other sections in other config files)?

Reply:
That's the whole point of this PR.  People shouldn't need to point to these directories manually now that the directory structure is standardized.

Are you thinking that people would need to make their own region masks and point to those directories?  I guess that's conceivable, particularly for non-standard grids.

My thought was that we want the example config files to be relatively simple.  I guess you prefer to keep them relatively verbose and to point out to users that they could point the `mappingSubdirectory` and/or `regionMaskSubdirectory` as an absolute path?

Reply:
No, in this case I only meant the section below that has other (unrelated) things in it.
I would only keep config.example more verbose.

Reply:
I see.  The way comments work on GitHub, all I see are the previous 3 lines to your comment so that wasn't clear.   It could be that i need to put the MOC section back in but I don't think the rest of this is helpful.  It just makes it a lot harder to change default options because all of the example config files have to be changed to match.  I think the examples should be relatively lean.

Reply:
ok, agree that only the MOC section should be put back in for this specific config files, while the others are redundant.